features/step_definitions/steps.rb

```diff
@@ -945,7 +945,7 @@ def check_expanding_report(string)
 
 Given(/^we have these users in our database$/) do |table|
   table.hashes.each do |row|
-    create(:user, email: row['Email'])
+    create(:user, name: row['Email'])
```

Comment:
The row name could be changed to name, to prevent confusion, since the name and not the email is used anymore.